operatorconfig/moduleconfig/authorization/v1.4.0/ingress.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: proxy-server
+  namespace: <NAMESPACE>
+spec:
+  ingressClassName: <PROXY_INGRESS_CLASSNAME>
+  tls:
+  - hosts:
+      - <AUTHORIZATION_HOSTNAME>
+      - <PROXY_INGRESS_HOSTS>
+    secretName: karavi-auth-tls
+  rules:
+  - host: <AUTHORIZATION_HOSTNAME>
+    http:
+      paths:
+        - path: /
+          pathType: Prefix
+          backend:
+            service:
+              name: proxy-server
+              port:
+                number: 8080
+  - host: <PROXY_INGRESS_HOSTS>
```

Comment:
Is this a placeholder for multiple values for the proxy host? If there are multiple values, they would each need the rest of the specification of the backend service like this:

```
  - host: proxy-host-one
    http:
      paths:
        - path: /
          pathType: Prefix
          backend:
            service:
              name: proxy-server
              port:
                number: 8080
  - host: proxy-host-two
    http:
      paths:
        - path: /
          pathType: Prefix
          backend:
            service:
              name: proxy-server
              port:
                number: 8080
```